### ISSUE TITLE
New language training - Thai [Feature]

### ISSUE THREAD

**Issue by TanZhili** [fishaudio/fish-speech#207]
I am training the model for a new language, Thai.
1. Should I only modify fish_speech/text/clean.py  to allow the unicodes for thai? Is there any other codes that I should modify?
2. I use about 200 hours data to do fine-tuning on medium model/codebook and lora on large model/codebook. After thousands steps, both the models still cannot generate correct thai sounds. Should I train more steps?

Finetune logs:
Epoch 0: |          | 5267/? [16:46:54<00:00,  0.09it/s, v_num=2, train/loss=7.090, train/top_5_accuracy=0.209, val/loss=7.640, val/top_5_accuracy=0.172]

Lora logs:
Epoch 0: |          | 1242/? [16:41:25<00:00,  0.02it/s, v_num=5, train/loss=9.500, train/top_5_accuracy=0.154, val/loss=10.60, val/top_5_accuracy=0.115]

**Comment by leng-yue** [comment]
You only need to update the clean.py I believe. Did you observe overfitting during training? If not, you probably need to train for more steps.

**Comment by leng-yue** [comment]
Ideally 100k, but I am not sure what will happen on a 200 hours dataset... We didn't explore pretraining with small dataset currently. I will suggest to set a lower learning rate, like 4e-5, 2e-5, enable weight decay, and enable dropout to counter overfitting.

**Comment by maxbizz** [comment]
@TanZhili Do let us know how many steps and what settings you set, if you successfully train your thai model.

**Comment by NisargBhavsar25** [comment]
@TanZhili Were you able to train your Thai model successfully? If yes can you please share your training configs?
